private/pkg/app/appcmd/webpages.go

```diff
@@ -0,0 +1,307 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package appcmd
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"html"
+	"io"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+	"unicode"
+
+	"github.com/bufbuild/buf/private/pkg/app"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"go.uber.org/multierr"
+)
+
+const (
+	slugPrefixFlagName      = "slug-prefix"
+	excludeCommandsFlagName = "exclude-command"
+)
+
+var codeBlockRegex = regexp.MustCompile(`(^\s\s\s\s)|(^\t)`)
+
+type webpagesFlags struct {
+	SlugPrefix      string
+	ExcludeCommands []string
+}
+
+func newFlags() *webpagesFlags {
```

Comment:
`newWebpagesFlags`

Reply:
Resolved in 7d7ba75363b90cbe2167c1d5100982451578644d